content/en/docs/components/pipelines/v2/run-a-pipeline.md

```diff
@@ -1,80 +1,107 @@
 +++
 title = "Run a Pipeline"
-description = "Execute a pipeline on the KFP backend"
+description = "Run a pipeline"
 weight = 6
 +++
 
-The KFP SDK offers three ways to run a pipeline.
+You can run a pipeline in three ways:
 
-## 1. Run from the KFP Dashboard
-The first and easiest way to run a pipeline is by submitting it via the KFP dashboard.
+* [Run a pipeline from the KFP Dashboard](#run_from_dashboard)
```

Comment:
nit: can maybe remove "Run a pipeline" prefix from each of these for brevity

Reply:
My suggestion would be to remove "a pipeline" and retain "Run".